src/inference/src/dev/core_impl.cpp

```diff
@@ -628,7 +790,10 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::compile_model(const std::string& mod
     if (cacheManager && device_supports_model_caching(plugin)) {
         CacheContent cacheContent{cacheManager, model_path};
         cacheContent.blobId = ov::ModelCache::compute_hash(model_path, create_compile_config(plugin, parsed._config));
-        auto lock = cacheGuard.get_hash_lock(cacheContent.blobId);
+        std::unique_ptr<CacheGuardEntry> lock;
+        // Proxy plugin fallback to lowlevel device
```

Comment:
maybe proxy need to support import / export ?
In this case we will avoid this condition.

Reply:
It is not possible for all cases. Proxy emulates low level plugin behaviour